pmd-java/src/test/resources/net/sourceforge/pmd/ant/xml/pmdtasktest.xml

```diff
@@ -143,4 +143,22 @@
             </fileset>
         </pmd>
     </target>
+
+	<target name="testAnalysisCache">
+		<pmd rulesetfiles="src/main/resources/category/java/design.xml/ExcessiveMethodLength" cacheLocation="/home/user/.pmd/cache">
```

Comment:
this cache-location assumes *nix-like OS... I'd rather have a relative file (ie: `pmd.cache`) than an absolute path